activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/validation/extensions/TaskMappingsServiceTaskImplementationValidator.java

```diff
@@ -109,6 +109,11 @@ public TaskMappingsServiceTaskImplementationValidator(ConnectorModelType connect
                 .map(Optional::get);
     }
 
+    private boolean isNotConnector(Optional<String> implementationTask) {
+        return implementationTask.isEmpty() || (implementationTask.get().equals("dmn-connector.EXECUTE_TABLE") && implementationTask.get()
```

Comment:
should not be an OR in between dmn and script check? 🤔

Reply:
ah yes I thought I force pushed the commit fixing it, nice spot!

Reply:
done